### PR TITLE
Remove Terms and Conditions

### LINKED ISSUE #53360: [FE] (Tech Debt) - Remove Terms & Conditions from vets-website
## Description
Remove the old Terms & Conditions functionality from vets-website and ensure any code redirect to that page is also removed.

## Tasks
- [x] Remove the `terms and conditions` application from vets-website
- [x] Notify in vfs-frontend & vfs-all-teams that Terms & Conditions application will be removed
- [x] Ensure `cta-widget` that handles some MHV stuff does not redirect to terms & conditions

### PR DESCRIPTION
**Pull request by ianmagenta**
## Summary
Remove the old Terms & Conditions functionality from vets-website and ensure any code redirect to that page is also removed.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/53360

## Acceptance criteria
- [x] Remove the `terms and conditions` application from vets-website
- [x] Notify in vfs-frontend & vfs-all-teams that Terms & Conditions application will be removed
- [x] Ensure `cta-widget` that handles some MHV stuff does not redirect to terms & conditions

### CODE REVIEW

File: src/applications/static-pages/cta-widget/index.js [hunk 1]
```diff
@@ -367,24 +366,11 @@ export class CallToActionWidget extends Component {
...
 
-    const redirectToTermsAndConditions = () => {
-      const redirectQuery = { tc_redirect: window.location.pathname }; // eslint-disable-line camelcase
-      const termsConditionsUrl = appendQuery(
-        '/health-care/medical-information-terms-conditions/',
-        redirectQuery,
-      );
-      window.location = termsConditionsUrl;
-    };
-
```
Comment (asg5704) [inline src/applications/static-pages/cta-widget/index.js:378]: 🚀

**Comment by asg5704** [comment]
Looks good, can you also create a PR to remove the app from the registry as well. Thank you!
https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json

**Review by asg5704 — APPROVED** [review]
Looks good, can you also create a PR to remove the app from the registry as well. Thank you!
https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json

**Comment by adamwhitlock1** [comment]
We are currently linking to this application in the Profile application, and also showing specific information in account security based on the terms being accepted or not.

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/personalization/profile/components/account-security/MHVTermsAndConditionsStatus.jsx#L7

Is there a replacement for this application, or should we just be removing this section of the Profile? 

As it stands we would definitely be displaying broken links/causing issues once this app is removed if we don't mitigate first.

CC @BerniXiongA6 and @Samara-Strauss

**Comment by asg5704** [comment]
@adamwhitlock1 As far as I am aware the Terms & Conditions will be replaced at some point but the current T&C was created as bridge for MHV accounts interfaces that are no longer required. The section should probably be removed from the Profile until it is rebuilt. I have a PR that addresses the other parts of MHV Accounts creation & create and upgrade here https://github.com/department-of-veterans-affairs/vets-website/pull/23438.

**Comment by jilladams** [comment]
Looks like Public Websites FE is marked as a required reviewer due to changes under /src/applications/static-pages ([codeowner#L73](https://github.com/department-of-veterans-affairs/vets-website/blob/main/.github/CODEOWNERS#L73)). 

The ticket for this PR (#53360) has no info on the business reasons for making the change, and it's in a general Identity Tech Debt epic, so I don't feel like we know enough to sign off on it, esp given Adam's feedback above. Platform COP Frontend is also marked as a required reviewer, so we'd like to let Platform FE be the ones to sign off on this unless folks have any objections.

**Comment by Samara-Strauss** [comment]
I asked in Slack as well, but has this been resolved?

**Comment by Samara-Strauss** [comment]
[Lauren Alexanderson just posted this in slack](https://dsva.slack.com/archives/CE4304QPK/p1677590994027809?thread_ts=1677099088.687269&cid=CE4304QPK):

> Ultimately, we should not touch anything on T&C in [VA.gov](http://va.gov/) until the new T&C pathway is there, and then we do need to show users in profile that they have signed it, and give them a way to revoke signing it.

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request closed without merging**